### ISSUE TITLE
Test set and Validation set

### ISSUE THREAD

**Issue by Lelo123** [harvitronix/five-video-classification-methods#95]
Good night.
I'm a little bit new to Deep Learning but isn't it necessary to diferenciarem between Validation and test set?

**Comment by aurotripathy** [comment]
If you are asking about the train and validation sets,  you may have to follow this code. 
```
 50    if load_to_memory:
 51        # Get data.
 52        X, y = data.get_all_sequences_in_memory('train', data_type)
 53        X_test, y_test = data.get_all_sequences_in_memory('test', data_type)
 54    else:
 55        # Get generators.
 56        generator = data.frame_generator(batch_size, 'train', data_type)
 57        val_generator = data.frame_generator(batch_size, 'test', data_type)
 58

```
If you are asking about inference-time testing with test set, that has not been provided.
